### PR TITLE
start_transfer: log exceptions

### PR DESCRIPTION
**Pull request by mistydemeo**
Fixes #8117

### CODE REVIEW

File: src/dashboard/src/components/filesystem_ajax/views.py [hunk 1]
```diff
@@ -255,6 +255,7 @@ def start_transfer(transfer_name, transfer_type, accession, paths, row_ids):
                 type=transfer_type, accession=accession,
                 transfer_metadata_set_row_uuid=row_id)
         except Exception:
+            logging.exception("Error copying %s to start of transfer", filepath)
```
Comment (Hwesta) [inline src/dashboard/src/components/filesystem_ajax/views.py:258]: Do we want this to show up as ERROR level?  Is `logging.info(..., exc_info=True)` enough?

Comment (mistydemeo) [inline src/dashboard/src/components/filesystem_ajax/views.py:258]: I think we'd want this to usually be printed; at `info` there's a good chance that many installations would be configured to not show it.

Comment (Hwesta) [inline src/dashboard/src/components/filesystem_ajax/views.py:258]: Good point, it is a pretty important error.

**Comment by Hwesta** [comment]
:+1:

**Pull request merged**